opencti-platform/opencti-graphql/src/utils/markingDefinition-utils.js

```diff
@@ -0,0 +1,86 @@
+import * as R from 'ramda';
+import { getEntitiesMapFromCache } from '../database/cache';
+import { ENTITY_TYPE_MARKING_DEFINITION } from '../schema/stixMetaObject';
+import { SYSTEM_USER } from './access';
+import { UPDATE_OPERATION_ADD, UPDATE_OPERATION_REMOVE, UPDATE_OPERATION_REPLACE } from '../database/utils';
+
+export const cleanMarkings = async (context, values) => {
+  const markingsMap = await getEntitiesMapFromCache(context, SYSTEM_USER, ENTITY_TYPE_MARKING_DEFINITION);
+  const defaultMarkingValues = values?.map((d) => markingsMap.get(d) ?? d) ?? [];
+  const defaultGroupedMarkings = R.groupBy((m) => m.definition_type, defaultMarkingValues);
+  return Object.entries(defaultGroupedMarkings).map(([_, key]) => {
+    const max = Math.max(...key.map((m) => m.x_opencti_order));
+    const results = key.filter((m) => m.x_opencti_order === max);
+    return R.uniqWith((a, b) => a.id === b.id, results);
+  }).flat();
+};
+
+// eslint-disable-next-line @typescript-eslint/default-param-last
+export const handleMarkingOperations = async (context, currentMarkings = [], refs, operation) => {
+  try {
+    // Get all marking definitions
```

Comment:
It's not a good idea to disable es lint
You can have a new property and handle the null case

```suggestion
    // Get all marking definitions
const markings = currentMarkings ?? []
```

Reply:
Done, thanks :)